airbyte-integrations/bases/connector-acceptance-test/Dockerfile

```diff
@@ -12,8 +12,7 @@ RUN apt-get update \
 RUN curl -fsSL https://get.docker.com | sh
 
 
-
-RUN pip install poetry==1.5.1
+RUN pip install poetry==1.8.4
```

Comment:
We might consider poetry 2.0 here, but no reason I know of that would cause an issue.

Reply:
Yeah, just playing it safe.